Create podcast blogpost (#102)
publiccodenet/blog: _posts/2020-09-01-on-air-next-our-podcast.md

---
title:  "On air next: our podcast!"
date: 2020-09-01
author: Alba Roza
type: blogpost

> excerpt: Launch of the livestream/videopodcast project

Reviewer: To get the name everywhere the excerpt goes :)

```suggestion
excerpt: We're launching a new livestream/video podcast: 'Let's talk about public code'
```

category:
  - Codebase stewardship
  - Communications
---

# On air next: our podcast!

A community couldn't exist without knowledge sharing. Indeed, this is one of its most powerful and appealing characteristics, although it's obvious that not all of its members can invest the same number of hours collaborating in a project. 

Sharing assets is something we always keep in mind at the Foundation for Public Code. We've always tried to share as many guides, tools and tips as we can in order to nurture our community. However there was something we were lacking, an even more direct way of communicating with our community: a live, closer and bilateral communication.

This is why we are **launching our first live stream/podcast, 'Let's talk about public code'**, a space in which we'll interview people strongly related to popular public and open codebases, 'public code' ones.

> ![Listen to our podcast in our Youtube channel]({{site.url}}/assets/podcast-available-youtube-channel.jpg)

Reviewer: Shouldn't this link to one/all of the livestream platforms, so people can pick their fav? That way it makes more sense to say later on that you can catch it on YouTube if you didn't make it.
(I assume most people consider YouTube to be for pre-recorded content, so we should make extra clear where/how people can view the live version.)

Author: You're right. Let's add it in the end of the blog post.

## What? When? How?!

Stay tuned next **Thursday, 8 September at 14.00 (UTC)** for the first episode. You will be able to ask real time questions of our first guest, Dr. Arnau Monterde, coordinator from the popular project [Decidim](https://decidim.org/), a public codebase that has been replicated in dozens of organizations in Spain, Canada, Finland, Mexico, France, Sweden and Italy.

> Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Come along and [watch our YouTube channel](https://www.youtube.com/channel/UCXIL94kkenw0cs_ZgNhKYuw) next Thursday, we'll be waiting for you!

Reviewer: ```suggestion
Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Bring all your questions to our livestream next Thursday, we'll be waiting for you!
```

> Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Bring all your questions to our livestream next Thursday, we'll be waiting for you! We are going to be live in several platforms like [Youtube](https://www.youtube.com/channel/UCXIL94kkenw0cs_ZgNhKYuw), [Periscope](https://twitter.com/publiccodenet), [Facebook](https://www.facebook.com/publiccodenet) and [Linkedin](https://www.linkedin.com/company/publiccodenet/), choose your favourite one and join the conversation! 

Author: I've added the different channels here, @ElenaFdR

> Are you unable to join then? Don't worry, we've also thought about that. We are going to publish the interview in [podcast format](https://podcast.publiccode.net/) once it's edited, so everyone has the chance of listening to Arnau's insights.

Reviewer: ```suggestion
Are you unable to join then? Don't worry, we've also thought about that. We'll publish the interview in [podcast format](https://podcast.publiccode.net/) once it's edited, so everyone has the chance of listening to Arnau's insights.
```
